generators/root/templates/base/terraform/terraform.tfvars.example

```diff
@@ -0,0 +1,3 @@
+scw_access_key = "{your scw_access_key}"
+scw_secret_key = "{your scw_secret_key}"
+scw_project_id = "{your scw_project_id}"
```

Comment:
j'avais mis zone et region dans mon tfvars je ne sais plus pk

Reply:
(c'est pas déconnant de ne pas les mettre)

Reply:
I think requiring them would make the project setup a bit more complex and people can still change them later on if needed.